walkthrough.md

```diff
@@ -69,6 +66,29 @@ To scan another project, press the Previous button and select a different projec
 
 Press Next if you're done scanning projects.
```

Comment:
I think this is missing the part where we produce the json. Suggesting an approach similar to https://github.com/GoogleCloudPlatform/inspec-gcp-cis-benchmark/pull/59 where the json filename is dynamic based on the project being scanned. Will need a refactor of mentions of `myscan.json` further in this doc.